### ISSUE TITLE
enhancement: Create pipeline as a GitHub Action

### ISSUE THREAD

**Issue by tobybaratta** [sinaatalay/rendercv-pipeline#2]
Because one might clone this repository and use it for creating their CV, it might be good to explicitly have this be a GitHub Action, so that this can be clearly versioned for users and they're able to get a notification/warning when they need to update the Action. 

This would make sure that folk don't run into issues when using the webapp (on the latest version), then aren't able to use this pipeline anymore without re-cloning this repository.

Might be fairly simple? Not sure how the GitHub Actions pipeline works, versus templates. Or how that setup would run, I suppose.

**Comment by sinaatalay** [comment]
You are right; this is definitely necessary. I will comment on this issue and close it once it is resolved.
